lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -3857,6 +3857,9 @@ bool TypeSystemSwiftTypeRef::ShouldSkipValidation(opaque_compiler_type_t type) {
   if (mangled_name == "$sSo11CFStringRefaD")
     return true;
 
+  if (mangled_name == "$sSo7CGPointVD")
+    return true;
+
```

Comment:
Looks like CGSize also trips the same assertion, and I'm guessing there could be more.

```suggestion
  if (mangled_name == "$sSo7CGPointVD")
    return true;

  if (mangled_name == "$sSo6CGSizeVD")
    return true;
```

Reply:
Actually, the test succeeded with just those two.